docs/ref/models/querysets.txt

```diff
@@ -1981,15 +1983,19 @@ evaluated will force it to evaluate again, repeating the query.
 Also, use of ``iterator()`` causes previous ``prefetch_related()`` calls to be
 ignored since these two optimizations do not make sense together.
 
-.. warning::
+.. note::
+
+    Some Python database drivers perform caching using client side cursors
+    (instantiated with ``connection.cursor()`` and what Django's ORM uses) and
+    cannot stream results from the database. The following database engines
```

Comment:
I'm not sure this first sentence adds much value. I know it's adapted from the previous documentation, but it reads strange to me, especially since the previous advice (about drivers doing their own caching) seemed to only apply to psycopg2 which is no longer relevant.

How about replacing the first sentence with something like:

> Some Python database drivers will still load the entire result set into memory, but won't cache results after iterating them. The following database engines support streaming results from the database without loading the entire result set into memory (using server-side cursors):

@timgraham what do you think about this wording? As far as I'm concerned the code is fine, but docs might need some light touches, which is why I haven't marked it Ready for Checkin yet.